src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerHost.cs

```diff
@@ -47,7 +47,7 @@ internal sealed partial class DesignerHost : Container, IDesignerLoaderHost2, ID
         private BitVector32 _state; // state for this host
         private DesignSurface _surface; // the owning designer surface.
         private string _newComponentName; // transient value indicating the name of a component that is being created
-        private Stack _transactions; // stack of transactions.  Each entry in the stack is a DesignerTransaction
+        private Stack<DesignerTransaction> _transactions; // stack of transactions.  Each entry in the stack is a DesignerTransaction
```

Comment:
I think the second part of the comment can be removed now
```suggestion
        private Stack<DesignerTransaction> _transactions; // stack of transactions
```